packages/components/popper/src/content.vue

```diff
@@ -113,6 +113,15 @@ export default defineComponent({
           immediate: true,
         }
       )
+
+      watch(
+        () =>
+          buildPopperOptions(props, {
+            arrowEl: unref(arrowRef),
+            arrowOffset: unref(arrowOffset),
+          }),
+        (option) => popperInstanceRef.value?.setOptions(option)
+      )
```

Comment:
I think this is smart, this should work just fine.

Have you tested this in your local machine with the test test case?

Reply:
Of course~

![image](https://user-images.githubusercontent.com/11560552/153149026-d28e84b9-6305-4beb-bc90-584a508f621a.png)
![image](https://user-images.githubusercontent.com/11560552/153149099-2ed1c644-7430-446d-8e5b-cd97996cf43e.png)

Reply:
I had tested the case #5838. It worked.